src/common/api/types.ts

```diff
@@ -922,7 +922,8 @@ export interface FeatureSupportLevel {
       | 'SINGLE_NODE_EXPANSION'
       | 'LVM'
       | 'DUAL_STACK_NETWORKING'
-      | 'MULTIARCH_RELEASE_IMAGE';
+      | 'MULTIARCH_RELEASE_IMAGE'
+      | 'NUTANIX_INTEGRATION';
```

Comment:
Not sure if we should check this or it's implicit from the `supported-platforms` API response.